### PR TITLE
Blockscout chart refactor: affinity. Resources updated

### PR DESCRIPTION
**Pull request by jcortejoso**
### Description

Refactor blockscout chart:
- Format and best practices
- Added `affinity` Values.
- Updated `alfajores` resource usage based on [metrics](https://clabs.grafana.net/goto/eQjobabVz?orgId=1) and using affinity to better pod allocation.

### Tested

Deployed in testnets

### CODE REVIEW

File: packages/helm-charts/blockscout/values-alfajores-blockscout2.yaml [hunk 1]
```diff
@@ -30,16 +40,41 @@ blockscout:
...
+              labelSelector:
+                matchExpressions:
+                  - key: release
+                    operator: In
+                    values:
+                      - alfajores-blockscout2
+                  - key: component
+                    operator: In
+                    values:
+                      - blockscout-api
```
Comment (carterqw2) [inline packages/helm-charts/blockscout/values-alfajores-blockscout2.yaml:76]: Why do both api and web pods have the same anti-affinity rule?

File: packages/helm-charts/blockscout/values.yaml [hunk 2]
```diff
@@ -314,3 +320,6 @@ blockscout:
   domainName: celo-testnet.org
   gcp:
     projectId: celo-testnet-production
+
+affinity: {}
```
Comment (carterqw2) [inline packages/helm-charts/blockscout/values.yaml]: Wrong indentation. Looking at the [template](https://github.com/celo-org/celo-monorepo/pull/10209/files#diff-06961cdf1f8a0aefca30f5075472e255d9a6a52449f7c89148cc4af4e93bb8e4R179) you need it to be under `blockscout`. Let's maybe also put it under some meaningful name like `k8s` or `k8sDefaults` or something, now it's a little hard to understand the scope of these values.

File: packages/helm-charts/blockscout/values-alfajores-blockscout2.yaml (hunk 1 above)
Comment (jcortejoso) [reply packages/helm-charts/blockscout/values-alfajores-blockscout2.yaml:76]: Good seen! Thanks

File: packages/helm-charts/blockscout/values.yaml (hunk 2 above)
Comment (jcortejoso) [reply packages/helm-charts/blockscout/values.yaml]: Thanks! Indeed I forget to save this file 🙃 .

Regarding Values "refactor", when moving to `charts` repository, another task we do is formatting the values, document and generate a Readme, etc... I think that moment is a good opportunity to current structure and adapt to something that makes more sense (also from a documentation/organization point of view).

**Review by carterqw2 — APPROVED** [review]
Looks good!

**Pull request merged**